### DISCUSSION TITLE
Include other Docker containers

### DISCUSSION THREAD

**Discussion by casparjones** [caprover/caprover#1260] [Ideas]
Hello Caprover Community,
I discovered Caprover a few weeks ago and am very excited about how easy and convenient everything works. I have now installed Caprover in an existing Docker project and everything worked. I was then just wondering if it might be a profitable idea to be able to make former Docker images/containers visible in Caprover, then you would have an overview in Caprover.
For that, I might want a tab in the app overview for apps with frontend/URL (open) and containers for data without URL. That would probably improve the overview a bit more.
What do you think about the ideas?
Greetings Frank

**Comment by githubsaturn** [comment]
>  idea to be able to make former Docker images/containers visible in Caprover, then you would have an overview in Caprover.

What do you mean by former Docker images/containers? Are you referring to containers that are deployed outside of CapRover?

**Reply by casparjones** [reply]
Exactly, I had already often used Docker and Composer-docker in the system and therefore still database containers, which I have now connected to the new Caprover Docker containers to be able to access the old data. I am really excited about the simplicity of Caprover and would like to use it only and would have liked a way to include the old DB containers in Caprover.

**Reply by githubsaturn** [reply]
There is a number of good tools for this purpose. Portainer is the best one. You can actually install Portainer as a one-click app from CapRover. 

Portainer is just a nice GUI for docker, while CapRover manages your domains and etc for you. You can use them together.

**Reply by casparjones** [reply]
Yes, thanks for the tip! Portainer I have also already installed.
Specifically, this is about Mattermost. I had installed Mattermost via the one-click installation in caprover and then connect the mattermost app to the old database container (which I previously fetched into the caprover network via portainer). 

This also worked wonderfully, must admit I'm not that deep into the Docker world yet. Unfortunately, I had never managed to connect via networks using the CLI, rest thanks to Portainer and Caprover I have the sense of achievement. ;-) But to keep the overview, I had thought then, that it is a pity not to make the old DB container visible in Caprover 🤔  or is the thought a bit absurd? Probably a rather rarely used edge case, that's probably true.

**Reply by katistix** [reply]
Hi! I have a container already running for some time and recently installed caprover. Can I somehow add that (already existing) container into my caprover dashboard?

**Reply by githubsaturn** [reply]
Responded here https://github.com/caprover/caprover/issues/1450

**Reply by katistix** [reply]
Thanks!
